src/Actions/WheelMUD.Actions.csproj

```diff
@@ -35,4 +35,7 @@
     <Compile Remove="Inform\Uptime.cs" />
     <Compile Remove="Inform\Version.cs" />
   </ItemGroup>
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```

Comment:
I found the method I was trying to think if we still had - `EnsureFilesArePresent()` in Application.cs - it tries to ensure we have the Files directory and if not present, copies it over.  It should be doing what this tries to do but at app startup.

Could you delete your data copy of Files and debug what's going on in that function instead?  I'd prefer if we could make that more reliable over build-time automation.  We tried build automation for this in the past and it was unfortunately frail.  (The commands here might exist on every OS we support today by default, but at minimum, we don't want to produce warning/error output if the directory already exists, and we don't want to stomp over existing Files contents if it already exists.  When we add multi-line in-game editing, like for OLC and message boards - we might also want to let admins edit the credits directly in-game.  Basically, credits.txt and the like becomes game data after that first copy that we don't want to stomp it, much like seeding any other database or other data with an initial version.)

Reply:
Perhaps the copying is working?  Also the copying is supposed to be moving over the MUD splash screens, so if you're getting far enough to log in, it seems the copy step must be working.  So perhaps look at `LoadSplashScreens()` to see how that loads those in a safer manner, and adjust to follow that pattern for Credits to see if it solves the loading bug?

Reply:
You're right. The credits file is being copied into C:\ProgramData\WheelMUD\WheelMUD\Files but the credits command class is looking for credits.txt in C:\code\WheelMUD\binDebug\netcoreapp3.1\Files.

Should I point the Credits command to the ProgramData folder instead? If so just ignore my last commit. I started changing things and checked in a fix before I read your last comment.

Reply:
We should be loading the Credits file via the ProgramData.  So since it's getting copied there successfully, we probably just need to adjust the StreamReader path to load it from the right path.